plugins/modules/proxmox_cluster_ha_groups.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2025, Markus Kötter <koetter@cispa.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_cluster_ha_groups
+
+short_description: Management of HA groups in roxmox VE Cluster
```

Comment:
I think a P is missing here :)
```suggestion
short_description: Management of HA groups in Proxmox VE Cluster
```